commit 4fc298b86635c60061bbd81cef8de2b031e5c4a7
Author: Christian Lamparter <chunkeey@web.de>
Date:   Tue Mar 24 21:58:08 2009 +0100

    ar9170: add Cace Airpcap NX usb_id
    
    This patch adds a new device to ar9170usb.
    
    Reported-by: Mike Kershaw/Dragorn <dragorn@kismetwireless.net>
    Signed-off-by: Christian Lamparter <chunkeey@web.de>
    Signed-off-by: John W. Linville <linville@tuxdriver.com>

diff --git a/drivers/net/wireless/ar9170/usb.c b/drivers/net/wireless/ar9170/usb.c
index ad29684..43e8d8d 100644
--- a/drivers/net/wireless/ar9170/usb.c
+++ b/drivers/net/wireless/ar9170/usb.c
@@ -59,6 +59,8 @@ static struct usb_device_id ar9170_usb_ids[] = {
 	{ USB_DEVICE(0x0cf3, 0x9170) },
 	/* Atheros TG121N */
 	{ USB_DEVICE(0x0cf3, 0x1001) },
+	/* Cace Airpcap NX */
+	{ USB_DEVICE(0xcace, 0x0300) },
 	/* D-Link DWA 160A */
 	{ USB_DEVICE(0x07d1, 0x3c10) },
 	/* Netgear WNDA3100 */
